### DISCUSSION TITLE
Azure Blob  Storage

### DISCUSSION THREAD

**Discussion by VedikaGujar-GSLab** [dodona-edu/dolos#1064] [Q&A] [answered]
Hi All,

Can I pass azure blob storage files to Dolos instead of passing the file path from local machine?

So currently, I am storing all my files in Azure Blob Storage, instead of the local machine.
I wanted to pass files to Dolos from Azure Blob Storage, so the file path will be "https://.../test_128005_52115_.c".

Can I achieve the same?

**Comment by rien** [comment, marked as answer]
Dolos itself currently does not support passing URL's directly.

However, you can download the files to a temporary directory first.
